packages/odyssey-storybook/src/components/odyssey-mui/Typography/Typography.stories.tsx

```diff
@@ -129,6 +144,13 @@ export const TypographyStory: StoryObj<TypographyProps> = {
     children: "This is standard text.",
     variant: "body",
   },
+  render: (args) => {
+    const { variant, ...props } = args;
+    return React.createElement(
+      variantMapping[variant as TypographyVariantValue],
+      props as TypographyProps
+    );
+  },
```

Comment:
I'd like to import `createElement` directly for consistency with the rest of our imports.

```suggestion
  render: (args) => {
    const { variant, ...props } = args;
    return createElement(
      variantMapping[variant as TypographyVariantValue],
      props as TypographyProps
    );
  },
```